content/windows.md

```diff
@@ -563,11 +396,44 @@ Depending on how you installed Windows and which firmware your computer is runni
 - Press the `+` key (or `SHIFT` + `=`) to increase the number of seconds the system will wait before booting the default OS; press `-` to decrease the number of seconds.
 - Once a timer is set, the menu will display on every boot without having to hold down `SPACE`. (To reverse this, press `-` until the timer is back to zero.)
 
-## Repairing the Pop!_OS Bootloader
+##### Add Windows to systemd-boot
+
+If you installed Windows on a [dedicated drive](#installing-on-a-dedicated-drive), then the systemd-boot menu will not recognize the Windows drive right away. To make systemd-boot aware of the Windows installation, we will need to copy the Windows boot files from the Windows boot partition, to the Pop!\_OS boot partition.
+
+1. Boot to Pop!_OS
+2. Find the Windows drive
+    1. Run lsblk in a Terminal to list partitions on your systems
+        - The EFI partition should be the 1st partition on the Windows drive, and will be around 100M in size
+    2. Now mount the Windows EFI partition to /mnt
+        - sudo mount /dev/NAME where NAME is from the "name" column of the lsblk output for the EFI partition
+        - After mounting on a test machine, I ran ls -al /mnt/EFI and I was able to see a "Boot" and a "Microsoft" folder at this point
```

Comment:
Since we place directories in backtics, we should probably use that same formatting when referring to folders.